### PR TITLE
Adding trusted certificate authorities

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Content based on https://help.stackrox.com/docs/configure-stackrox/add-trusted-ca/

Preview https://openshift-docs-git-add-cert-auth-gnelson.vercel.app/openshift-acs/master/configuration/add-trusted-ca.html

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@bergerhoffer PTAL

File: modules/configure-additional-cas.adoc [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+[NOTE]
+====
+* You must use a PEM-encoded certificate file (with any extension).
+* You can also use the `-u` (update) option along with the `-f` option to update any previously added certificate.
+====
+.. Multiple certificates at once, move all certificates in a directory, and then use the `-d` (directory) option:
++
+[source,bash]
+----
+./ca-setup.sh -d <directory-name>
```
Comment (gaurav-nelson) [inline modules/configure-additional-cas.adoc:43]: Note to self: Add missing `$`

File: modules/configure-additional-cas.adoc [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+// * dir/filename.adoc
+:_module-type: PROCEDURE
+[id="configure-additional-cas_{context}"]
+= Configuring additional CAs
+
+To add custom CAs:
+
+.Procedure
+
+. Download the link:https://help.stackrox.com/downloads/927d7f5b23e88089b5545450cbfa7cc1/ca-setup.sh[_ca-setup.sh_] script.
```
Comment (gaurav-nelson) [inline modules/configure-additional-cas.adoc:12]: @IAmATeaPot418 Where are we planning to host this file? 
- And other files in `src/files` folders from our StackRox docs repository?

File: configuration/add-trusted-ca.adoc [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+//Restart Central
+include::modules/restart-central-container.adoc[leveloffset=+2]
+
+//Restart Scanner
+include::modules/restart-scanner-container.adoc[leveloffset=+2]
+
+[IMPORTANT]
+====
+After you have added trusted CAs and configured Central, the CAs are included in any new sensor deployment bundles that you create.
```
Comment (bergerhoffer) [inline configuration/add-trusted-ca.adoc:44]: Should "sensor" be "Sensor"? (a few other few places in the PR, if so)

File: files/ca-setup.sh [hunk 4]
```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env bash
```
Comment (bergerhoffer) [inline files/ca-setup.sh:1]: I don't think I've ever seen us include an actual script in our docs repo that we reference. 

I'm guessing that this should probably be hosted somewhere else, but @vikram-redhat what are your thoughts on this?

File: modules/configure-additional-cas.adoc [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+----
++
+[NOTE]
+====
+* You must use a PEM-encoded certificate file (with any extension).
+* You can also use the `-u` (update) option along with the `-f` option to update any previously added certificate.
+====
+.. Multiple certificates at once, move all certificates in a directory, and then use the `-d` (directory) option:
++
+[source,bash]
```
Comment (bergerhoffer) [inline modules/configure-additional-cas.adoc:40]: source,terminal

File: modules/configure-additional-cas.adoc [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+----
+$ ./ca-setup.sh -d <directory-name>
```
Comment (bergerhoffer) [inline modules/configure-additional-cas.adoc:42]: <directory_name>

File: modules/configure-additional-cas.adoc [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+----
++
+[NOTE]
+====
+* You must use PEM-encoded certificate files with `.crt` extension.
+* Each file should only contain a single certificate.
```
Comment (bergerhoffer) [inline modules/configure-additional-cas.adoc:48]: should -> must?

File: modules/configure-additional-cas.adoc [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+.. Multiple certificates at once, move all certificates in a directory, and then use the `-d` (directory) option:
++
+[source,bash]
+----
+$ ./ca-setup.sh -d <directory-name>
+----
++
+[NOTE]
+====
+* You must use PEM-encoded certificate files with `.crt` extension.
```
Comment (bergerhoffer) [inline modules/configure-additional-cas.adoc:47]: with -> with a (or the)

File: modules/restart-scanner-container.adoc [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
+You can restart the Scanner container by deleting the pod.
+
+.Procedure
+
+* Run the following command to delete the Scanner pod:
+** On {ocp}:
++
+[source,terminal]
+----
+$ oc delete po -n stackrox -l app=scanner
```
Comment (bergerhoffer) [inline modules/restart-scanner-container.adoc:19]: We don't normally use the `po` abbreviation for pod in our docs. Can you update these commands to use `pod` or `pods` instead?

**Review by bergerhoffer — COMMENTED** [review]
A few minor things, but looks good!

File: files/ca-setup.sh (hunk 4 above)
Comment (gaurav-nelson) [reply files/ca-setup.sh:1]: Thanks @bergerhoffer 
I did check with @vikram-redhat before adding this file. 
@vikram-redhat  can you confirm?

Comment (vikram-redhat) [reply files/ca-setup.sh:1]: This is ok.

Comment (bergerhoffer) [reply files/ca-setup.sh:1]: Great, thanks!

**Comment by bergerhoffer** [comment]
Thanks, LGTM - merging.

**Pull request merged**